### ISSUE TITLE
Lab08

### ISSUE THREAD

**Issue by ykharitonova** [ucsb-cs9/w24#17]
* test the search for a car that is not the first one in the `cars` list: `does_car_exists()` does this already but we need to test it in the `_add_car()` itself, so that by the time they get to finding the car, we/they can be sure that the cars have been properly added 
* have a way to visualize or report how many cars are stored in a list for a specific node (e.g., if we have more than one Prius that has different years and prices, display all cars that have been saved)

**Comment by ykharitonova** [comment]
@krushnashah FYI -- these are my notes from last quarter.

**Comment by krushnashah** [comment]
@ykharitonova yes, I have kept this in mind and updated the autograder and the lab description accordingly
